content/docs/how-we-work/baseline/why-ref-baseline.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Why Reference Baseline?
+weight: 1
```

Comment:
Shall we try a `weight` of 0? Right now, this page appears second in the list. See https://deploy-preview-204--axelerant-engg-handbook.netlify.app/docs/how-we-work/baseline/.